package.json

```diff
@@ -218,7 +218,7 @@
 		"preui-test": "npm run compile",
 		"lint": "eslint src --ext ts",
 		"test": "node ./out/test/runTest.js",
-		"ui-test": "node --unhandled-rejections=warn-with-error-code out/ui-test/uitest_runner.js"
+		"ui-test": "node --unhandled-rejections=warn-with-error-code out/ui-test/uitest_runner.js out/ui-test/tests/**/*.test.js"
 	},
 	"devDependencies": {
 		"@theia-extension-tester/repeat": "^0.1.2",
```

Comment:
can we get rid of `@theia-extension-tester/repeat` dependency if we are using `Repeat` from `vscode-uitests-tooling` ?